transcom/mymove
# B 18170 H&AL Prime delete PPM shipment

Summary

Add log for the H&AL when a Prime deletes a PPM shipment.

Verification Steps for Reviewers

These are to be checked by a reviewer.


How to test

1. Access the office app
2. Login as a Prime user
3. Find a Move with existing PPM shipment or create a new one
4. Click the Delete Shipment button
5. Click Delete Shipment button on Are you Sure? modal
6. Switch roles to a QAE user
7. Search for the Move from above
8. Click on Move history and verify that you can see the Delete shipment in the log

Frontend
